android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragment.java

```diff
@@ -119,6 +126,26 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         });
     }
 
+    private void createTimePicker() {
+        Calendar calendar = Calendar.getInstance();
+        int currentHour = calendar.get(Calendar.HOUR_OF_DAY);
+        int currentMinutes = calendar.get(Calendar.MINUTE);
+
+        timePicker = new MaterialTimePicker.Builder()
+                .setInputMode(MaterialTimePicker.INPUT_MODE_KEYBOARD)
+                .setTimeFormat(TimeFormat.CLOCK_24H)
+                .setHour(currentHour)
+                .setMinute(currentMinutes)
+                .setTitleText(R.string.time_message_date_picker)
```

Comment:
@Cebox82 I tested it and now it will work now with 

```java
     timePicker = new MaterialTimePicker.Builder()
                .setInputMode(MaterialTimePicker.INPUT_MODE_CLOCK)
```